src/warnet/warnet.py

```diff
@@ -42,10 +44,17 @@ def __init__(self, config_dir):
 
 
     def __str__(self) -> str:
-        template = "\t%-8.8s%-25.24s%-25.24s%-25.24s%-18.18s\n"
-        tanks_str = template % ("Index", "Version", "Conf", "Netem", "IPv4")
+        # TODO: bitcoin_conf and tc_netem can be added back in to this table
```

Comment:
Could use tabulate to do this for us? No need to re-invent the wheel, and I think adding a dep for it is OK?

```sh
pip install tabulate
```

```python
from tabulate import tabulate
    
    def __str__(self) -> str:
        headers = ["Index", "Version", "Conf", "Netem", "IPv4", "LN", "LN IPV4"]
        tanks_data = []
        for tank in self.tanks:
        	tanks_data.append([tank.index, tank.version, tank.conf, tank.netem, tank.ipv4, tank.lnnode.impl if tank.lnnode is not None else None, tank.lnnode.ipv4 if tank.lnnode is not None else None])

        return tabulate(tanks_data, headers=headers, tablefmt="plain")
```

Reply:
sure but maybe in a separate clean up PR since there are a few tables through our RPC.

Reply:
Leaving as unresolved just so I remember to do it in the future.

Reply:
Opened https://github.com/bitcoin-dev-project/warnet/pull/201